package.json

```diff
@@ -28,8 +28,8 @@
     "test:eslint": "eslint --config ./.eslintrc.yaml scripts/**/*.js source/js/**/*.js source/js/components/**/*.jsx",
     "test:json": "ajv validate -s source/json/upcoming.schema.json -d source/json/upcoming.json && ajv validate -s source/json/highlights.schema.json -d source/json/highlights.json",
     "test:pulled-json": "ajv validate -s source/json/people.schema.json -d source/json/temp/people.json && ajv validate -s source/json/news.schema.json -d source/json/temp/news.json",
-    "test:pug": "pug-lint source/pug/**/*.pug",
-    "test:scss": "stylelint 'source/sass/main.scss' --syntax scss",
+    "test:pug": "pug-lint source/pug/partials/ source/pug/templates/ source/pug/partials/",
+    "test:scss": "stylelint source/sass/main.scss --syntax scss",
```

Comment:
`shx` was double quoting the argument on windows, resulting in `'\'source/sass/main.scss\''`